Gruntfile.js

```diff
@@ -19,9 +19,11 @@ module.exports = function(grunt) {
     },
 
     // Handle vendor prefixing
-    autoprefixer: {
+    postcss: {
       options: {
-        browsers: ['last 2 versions', 'ie 8', 'ie 9']
+        processors: [
+          require('autoprefixer-core')({ browsers: 'last 2 versions' }),
```

Comment:
you could make it this, and still have ie8 and ie9 in there

`require('autoprefixer-core')({ browsers: ['last 2 versions', 'ie 8', 'ie 9']` }),`